content/docs/1.6.0/nodes-and-volumes/volumes/volume-conditions.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Volume Conditions
+weight: 7
+---
+
+## Volume Conditions 
+
+Volume conditions describe the current status of a volume and potential issues that may occur.
+- `Scheduled`: All replicas were scheduled successfully. 
+    If Longhorn was unable to schedule any of the replicas, the condition is set to `false` and error messages are displayed. The condition is set to `true` when the setting [Allow Volume Creation With Degraded Availability](../../../references/settings#allow-volume-creation-with-degraded-availability) is enabled and at least one replica is scheduled.
+- `TooManySnapshots`: This specific volume has more than 100 snapshots.
+    Longhorn allows you to create a maximum of 250 snapshots for each volume. For more information about configuring the maximum snapshot count, see [Snapshot Space Management](../../../snapshots-and-backups/snapshot-space-management).
+- `Restore`: Longhorn is restoring the volume from a backup.
+- `WaitForBackingImage`: The replicas have not started because the backing images must first be synced with their disks.
+
+## Engine Conditions
+
+Engine conditions describe the current status of a engine and potential issues that may occur.
```

Comment:
```suggestion
Engine conditions describe the current status of an engine instance and potential issues that may occur.
```

Reply:
Are you are referring to the engine instance on the node the volume is attached to? If not, you can ignore this suggestion.

Reply:
not the instance but the resource in kubernetes
`$kubectl get lhe -n longhorn-system`  (get longhorn engine CR)

Reply:
fixed, thanks